.github/workflows/book.rb

```diff
@@ -0,0 +1,21 @@
+require './item'
+
+# the book class
+class Book < Item
+  attr_accessor :publisher
+
+  def initialize(id, publisher, cover_state)
+    super()

```

Comment:
Super should pass the arguments needed for an item to be created
Eg: `super(publish_date, id, archived: archived)`